src/services/datahub-client.ts

```diff
@@ -230,6 +230,8 @@ export class HttpDatahubClient implements DatahubClient {
         from: new Date(x.from),
         to: new Date(x.to),
         quantity: x.quantity,
+        withEmail: x.withEmail,
```

Comment:
what happen if withEmail is undefined or null?

Reply:
That's the idea. That this will be not defined, do you thinks it's best to set a value anyways?

Reply:
how is the behavior when don't have value?

Reply:
This value should be undefined, and it's handled in the component as in the other components.